topics/proteomics/tutorials/metaquantome-data-creation/tutorial.md

```diff
@@ -0,0 +1,645 @@
+
+> ### {% icon hands_on %} Hands-on: 
+>
+> 1. {% tool [Unipept](toolshed.g2.bx.psu.edu/repos/galaxyp/unipept/unipept/4.3.0) %} with the following parameters:
+>    - *"Unipept application"*: `peptinfo: Tryptic peptides and associated EC and GO terms and lowest common ancestor taxonomy`
+>        - *"Equate isoleucine and leucine"*: `Yes`
+>        - *"retrieve extra information"*: `Yes`
+>        - *"group responses by GO namespace (biological process, molecular function, cellular component)"*: `Yes`
+>        - *"allfields"*: `Yes`
+>    - *"Peptides input format"*: `tabular`
+>        - {% icon param-file %} *"Tabular Input Containing Peptide column"*: `out_file1` (output of **Filter** {% icon tool %})
+>        - *"Select column with peptides"*: `c1`
+>    - *"Choose outputs"*: ``
+>
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > There are two Unipept in this workflow, One for taxonomy and other for function. Please select all the output options from Unipept.
+>    {: .comment}
+>
+The JSON output from the Taxonomy can be visualized using the visualize option and Select the Unipept Taxonomyviewer. 
+>
+>
+{: .hands_on}
+
+## *Extracting EC values*
+
+> ### {% icon hands_on %} Hands-on: 
+The cut tool cuts out specific columns from the dataset. In this case, the cut tool is being used to extract columns 1 (peptide) and 3 (EC number) from the dataset peptinfo EC.tsv output. This is a manipulation tool for metaQuantome's convinience.
+>
+> 1. {% tool [Cut](Cut1) %} with the following parameters:
+>    - *"Cut columns"*: `c1,c3`
+>    - {% icon param-file %} *"From"*: `output_ec_tsv` (output of **Unipept** {% icon tool %})
+>
+{: .hands_on}
+
+
+## *Filtering confident peptides*
+
+Query Tabular is a tool that can load tabular data into a SQLite database. This step precedes UniPept, as a list containing the peptide sequences must be generated. In this step a list of gene ontology (GO) terms is being generated.
+
+> ### {% icon hands_on %} Hands-on: 
+>
+> 1. {% tool [Query Tabular](toolshed.g2.bx.psu.edu/repos/iuc/query_tabular/query_tabular/3.0.0) %} with the following parameters:
+>    - In *"Database Table"*:
+>        - {% icon param-repeat %} *"Insert Database Table"*
+>            - {% icon param-file %} *"Tabular Dataset for Table"*: `output_go_tsv` (output of **Unipept** {% icon tool %})
+>            - In *"Table Options"*:
+>                - *"Specify Name for Table"*: `Goterm`
+>                - *"Specify Column Names (comma-separated list)"*: `peptide,total_protein_count,go_term,protein_count,go_name,go_funct`
+>    - *"SQL Query to generate tabular output"*: `SELECT Goterm.*
+```
+FROM Goterm 
+WHERE ((1.0*Goterm.protein_count)/(1.0*Goterm.total_protein_count)) >= 0.05
+```
```

Comment:
Fix the SQL query here so that the SELECT statement is moved to within the formatting:
```
SELECT Goterm.*
FROM Goterm
WHERE ((1.0*Goterm.protein_count)/(1.0*Goterm.total_protein_count)) >= 0.05
```